src/components/KTreeList/KTreeItem.vue

```diff
@@ -80,30 +83,31 @@ const handleClick = () => {
 </script>
 
 <style lang="scss" scoped>
-@import '@/styles/variables';
 @import '@/styles/functions';
+@import '@/styles/tmp-variables';
 
 .k-tree-item {
   align-items: center;
-  background-color: var(--KTreeListItemUnselectedBackground, var(--white));
-  border: 1px solid var(--KTreeListItemUnselectedBorder, var(--grey-200));
-  border-radius: 8px;
-  color: var(--KTreeListItemText, var(--black-500));
+  background-color: var(--KTreeListItemUnselectedBackground, var(--kui-color-background, $kui-color-background));
+  border: $kui-border-width-10 solid var(--KTreeListItemUnselectedBorder, var(--kui-color-background-neutral-weaker, $kui-color-background-neutral-weaker));
```

Comment:
issue: cannot use a background token in a border property, need to use `color-border-*` tokens or hard-code

Reply:
Is this something we can catch with style lint? (more useful to protect against incorrect usage in the future than as an immediate need now)

Reply:
Yes; we have a PR with some rules that can be enforced but are not ready to publish the stylelint plugin until after the onsite meetup so we can prepare guidelines